dask_ml/preprocessing/data.py

```diff
@@ -28,6 +28,8 @@ class StandardScaler(skdata.StandardScaler):
     def fit(self, X, y=None):
         self._reset()
         attributes = OrderedDict()
+        if isinstance(X, dd.DataFrame):
```

Comment:
Could you check for dask or pandas DataFrame's here? Then the learned attributes will always be ndarrays.

Reply:
Sounds good, have added a check for pandas DataFrames